BasementRenovator.py

```diff
@@ -4318,21 +4291,37 @@ def dropEvent(self, evt):
         self.openWrapper(target)
         evt.acceptProposedAction()
 
+    FIXUP_PNGS = (
+        "resources/UI/Bitfont.png",
+        "resources/Entities/5.100.0 - Collectible.png",
+        "resources/Backgrounds/Door.png",
+        "resources/Backgrounds/DisabledDoor.png",
+        "resources/UI/FilterIcons.png",
+        "resources/UI/ShapeIcons.png",
+        "resources/UI/uiIcons.png",
+        "resources/Entities/questionmark.png" "resources/UI/ent-error.png",
+        "resources/UI/ent-warning.png",
+        "resources/UI/CurrentRoom.png",
```

Comment:
lol, would you mind doing this as a hardcoded list of folders? You can keep the individual files for entities, or just relocate those special files to a special location